### ISSUE TITLE
wrong doc on Open5GS tutorial Dockerized VoLTE Setup

### ISSUE THREAD

**Issue by gmaruzz** [herlesupreeth/docker_open5gs#92]
probably totally independent from here, but tutorial in https://open5gs.org/open5gs/docs/tutorial/03-VoLTE-dockerized/ at step 2 (build base docker images) say:
<pre>
cd ../kamailio_base
</pre>

that is wrong, it must say:
<pre>
cd ../ims_base
</pre>

**Comment by herlesupreeth** [comment]
You are right. That needs correction. Please open a PR in open5gs repo. Thank you for pointing it out
